src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -1,545 +1,563 @@
-                  >
-                    <TextContent key={`${pkg.name}`}>
-                      <span className="pf-c-dual-list-selector__item-text">
-                        {pkg.name}
-                      </span>
-                      <small>{pkg.summary}</small>
-                    </TextContent>
-                  </DualListSelectorListItem>
-                );
+          </ToolbarItem>
+          <ToolbarItem>
+            <Button variant="primary" isInline>
+              Refresh
+            </Button>
+          </ToolbarItem>
+          <ToolbarItem>
+            <ToggleGroup>
+              <ToggleGroupItem
+                text="Available"
+                buttonId="toggle-available"
+                isSelected={toggleSelected === 'toggle-available'}
+                onChange={handleFilterToggleClick}
+              />
+              <ToggleGroupItem
+                text="Selected"
+                buttonId="toggle-selected"
+                isSelected={toggleSelected === 'toggle-selected'}
+                onChange={handleFilterToggleClick}
+              />
+            </ToggleGroup>
+          </ToolbarItem>
+          {/*FOLLOW UP
+          <ToolbarItem>
+            {' '}
+            <ToggleGroup>
+              <ToggleGroupItem
+                text="Included repos"
+                buttonId="toggle-included-repos"
+                isSelected={toggleSourceRepos === 'toggle-included-repos'}
+                onChange={handleRepoToggleClick}
+              />
+              <ToggleGroupItem
+                text="All repos"
+                buttonId="toggle-all-repos"
+                isSelected={toggleSourceRepos === 'toggle-all-repos'}
+                onChange={handleRepoToggleClick}
+              />
+            </ToggleGroup>
+          </ToolbarItem>
+          */}
+          <ToolbarItem variant="pagination">
+            <Pagination
+              itemCount={transformedPackages.length}
+              perPage={perPage}
+              page={page}
+              onSetPage={handleSetPage}
+              onPerPageSelect={handlePerPageSelect}
+              isCompact
+            />
+          </ToolbarItem>
+        </ToolbarContent>
+      </Toolbar>
+      <Table variant="compact" data-testid="packages-table">
+        <Thead>
+          <Tr>
+            <Th />
+            <Th width={20}>Package name</Th>
+            <Th width={35}>Description</Th>
+            <Th width={25}>Package repository</Th>
+            <Th width={20}>Support</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {!searchTerm && toggleSelected === 'toggle-available' && (
+            <EmptySearch />
+          )}
+          {searchTerm && transformedPackages.length === 0 && <NoResultsFound />}
+          {searchTerm &&
+            transformedPackages.length >= 100 &&
+            handleExactMatch()}
+          {transformedPackages.length < 100 &&
+            transformedPackages
```

Comment:
This is such a big code block. Could we extract the sorting logic into a helper function to help improve readability?

Reply:
Moved the sorting logic outside of the block.